docs/en/developers/set-autoscale.md

```diff
@@ -1,162 +1,208 @@
-    
-    Benchmarking 127.0.0.1 (be patient)
-    Completed 1000 requests
+    - Name: express-server
+      Type: webservice
+      HEALTHY Ready: 1/1
+      Traits:
+        - ✅ autoscale: type: cron    replicas(min/max/current): 1/4/1
+      Last Deployment:
+        Created at: 2020-11-05 17:09:03 +0800 CST
+        Updated at: 2020-11-05T17:09:02+08:00
   ```
   
-5. Monitor the replicas of the workload, and see its replicas increase from one to four.
-
+  Wait till the time clocks `startAt`, and check again. The replicas become to two, which is specified as 
+  `replicas` in `vela.yaml`.
+  
   ```
-    $ vela status helloworld --svc frontend
-    About:
-    
-      Name:      	helloworld
-      Namespace: 	default
-      Created at:	2020-11-05 20:07:21.830118 +0800 CST
-      Updated at:	2020-11-05 20:50:42.664725 +0800 CST
-    
-    Services:
-    
-      - Name: frontend
-        Type: webservice
-        HEALTHY Ready: 1/1
-        Traits:
-          - ✅ autoscale: type: cpu     cpu-utilization(target/current): 5%/14%	replicas(min/max/current): 1/5/4
-        Last Deployment:
-          Created at: 2020-11-05 20:07:23 +0800 CST
-          Updated at: 2020-11-05T20:50:42+08:00
+  $ vela status testapp
+  About:
+  
+    Name:      	testapp
+    Namespace: 	default
+    Created at:	2020-11-10 10:18:59.498079 +0800 CST
+    Updated at:	2020-11-10 10:18:59.49808 +0800 CST
+  
+  Services:
+  
+    - Name: express-server
+      Type: webservice
+      HEALTHY Ready: 2/2
+      Traits:
+        - ✅ autoscale: type: cron    replicas(min/max/current): 1/4/2
+      Last Deployment:
+        Created at: 2020-11-10 10:18:59 +0800 CST
+        Updated at: 2020-11-10T10:18:59+08:00
   ```
   
-  Stop `ab` tool, and the replicas will decrease to one eventually.
-
+  Wait after the period ends, the replicas will be one eventually.
 
-## Scale workload by cron
+## Setting auto-scaling policy of CPU resource utilization
+Introduce how to automatically scale workloads by CPU resource utilization.
 
-1. Prepare Appfile:
+1. Prepare Appfile
 
-  ```bash
-    $ cat <<EOF > vela.yaml
-      name: testapp
-      
-      services:
-        express-server:
-          image: zzxwill/kubevela-appfile-demo:v1
-      
-          cmd: ["node", "server.js"]
-          port: 8080
-      
-          autoscale:
-            min: 1
-            max: 4
-            cron:
-              startAt:  "14:00"
-              duration: "2h"
-              days:     "Monday, Thursday"
-              replicas: "2"
-              timezone: "America/Seattle"
-    EOF
-  ```
-
-2. Deploy the application:
+  Modify `vela.yaml` as below. We add field `services.express-server.cpu` and change the auto-scaling policy
+  from cron to cpu utilization by updating filed `services.express-server.autoscale`.
+  
+  ```yaml
+  name: testapp
+  
+  services:
+    express-server:
+      image: oamdev/testapp:v1
+        
+      cmd: ["node", "server.js"]
+      port: 8080
+      cpu: "0.01"
```

Comment:
Is this by intention to give such small cpu quota? why?

Reply:
Yes, by intention. With my benchmark tool, I generated 1000 requests by 200 qps (more qps might lead to crash of the tool ), if more CPU cores is set, the threshold value could not be easily reached. In order to let the example to work, I set 0.01 (10m) cores for the workload.
````
ab -n 10000 -c 200 http://127.0.0.1/
```

Reply:
If the workload doesn't run such a simple business, we can set more CPU resource for it. Do I need to change another image for the workload to make `cpu` resource looks better?